src/main.rs

```diff
@@ -484,6 +489,17 @@ fn cache_zip_archive(cache_location: &mut PathBuf, zip_name: &str, buffer: &[u8]
     Ok(())
 }
 
+fn remove_file(args: &Args, program_path: &Path) -> Result<()> {
+    if !args.force_remove {
+        return Ok(());
+    }
+
+    println!("Removing binary at {program_path:?}");
```

Comment:
I'd suggest to output this diagnostics only when verbose mode is turned on as this info most probably is not needed for end-users.

Reply:
Adding a `--verbose` flag seems reasonable to me.